docs/src/getting-started/installation.md

```diff
@@ -1,103 +1,310 @@
+
+1. Download the release archive matching your CPU architecture
+   from https://docs.swmansion.com/scarb/download.html#precompiled-packages.
+2. Extract it to a location where you would like to have Scarb installed. We recommend `%LOCALAPPDATA%\Programs\scarb`.
+3. From this directory, get the full path to `scarb\bin` and add it to PATH.
+   See [this article](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/) for instructions on
+   Windows 10 and 11.
+
+To verify that Scarb was installed, run
 
 ```shell
-$ snfoundryup
+scarb --version
 ```
 
-See `snfoundryup --help` for more options.
+and verify that version is >= 2.7.0
+
+### Install Universal Sierra Compiler
+
+1. Download the release archive matching your CPU architecture
+   from https://github.com/software-mansion/universal-sierra-compiler/releases/latest. Look for package with `windows`
+   in the name e.g.
+   `universal-sierra-compiler-v2.3.0-x86_64-pc-windows-msvc.zip`.
+2. Extract it to a location where you would like to have Starknet Foundry installed. We recommend
+   `%LOCALAPPDATA%\Programs\universal-sierra-compiler`.
+3. From this directory, get the full path to `universal-sierra-compiler\bin` and add it to PATH.
+   See [this article](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/) for instructions on
+   Windows 10 and 11.
+
+To verify that Starknet Foundry was installed, run
+
+```shell
+universal-sierra-compiler --version
+```
+
+### Install Starknet Foundry
+
+1. Download the release archive matching your CPU architecture
+   from https://github.com/foundry-rs/starknet-foundry/releases/latest. Look for package with `windows` in the name e.g.
+   `starknet-foundry-v0.34.0-x86_64-pc-windows-msvc.zip`.
+2. Extract it to a location where you would like to have Starknet Foundry installed. We recommend
+   `%LOCALAPPDATA%\Programs\snfoundry`.
+3. From this directory, get the full path to `snfoundry\bin` and add it to PATH.
+   See [this article](https://www.architectryan.com/2018/03/17/add-to-the-path-on-windows-10/) for instructions on
+   Windows 10 and 11.
 
-To verify that the Starknet Foundry is installed correctly, run `snforge --version` and `sncast --version`.
+To verify that Starknet Foundry was installed, run
 
-### Installation via [asdf](https://asdf-vm.com/)
+```shell
+snforge --version
+```
 
-First, add the Starknet Foundry plugin to asdf:
+or
 
 ```shell
-$ asdf plugin add starknet-foundry
+sncast --version
 ```
 
-#### Common Error
+## Common Errors
+
+### No Version Set (Linux and MacOS Only)
 
 Users may encounter this error when trying to use `snforge` or `sncast` without setting a version:
 
 ```shell
 No version is set for command snforge
-Consider adding one of the following versions in your config file at starknet-foundry 0.32.0
+Consider adding one of the following versions in your config file at $HOME/.tool_versions
+starknet-foundry 0.32.0
```

Comment:
nit: Will the version here be somehow swapped for latest one? It's the first place which user visits while interacting with foundry, so I think we should try to have the newest version here.

Reply:
It's just an example error